AVeryBigSum.cs

```diff
@@ -0,0 +1,52 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    /*
+     * Complete the 'aVeryBigSum' function below.
+     *
+     * The function is expected to return a LONG_INTEGER.
+     * The function accepts LONG_INTEGER_ARRAY ar as parameter.
+     */
+
+    public static long aVeryBigSum(List<long> ar)
+    {
+        long sum = 0;
```

Comment:
use var keyword instead of long. Using var instead of explicit data types like long is the best practice

Reply:
I am getting this error in HackerRank

Solution.cs(29,20): error CS0266: Cannot implicitly convert type `long' to `int'. An explicit conversion exists (are you missing a cast?)

Reply:
Resolved it @narayanagunturi